spec/spec_helper.rb

```diff
@@ -4,9 +4,12 @@
 
 require 'simplecov'
 
+require 'pry'
```

Comment:
please move this after `SimpleCov.start` just to play safe

Reply:
Roger that :robot: